engine/source/elements/solid/solidez/szforc3.F

```diff
@@ -144,6 +144,7 @@ SUBROUTINE SZFORC3(ELBUF_TAB,NG      ,
 #include      "timeri_c.inc"
 #include      "scr18_c.inc"
 #include      "sms_c.inc"
+#include      "scr05_c.inc"
```

Comment:
Could you pass the variable you need by argument?

Reply:
done following the comments.